src/views/waypoint/home/server.ts

```diff
@@ -0,0 +1,85 @@
+import slugify from 'slugify'
+// TODO: need to discuss from whence we should
+// TODO: source content down the road. For now,
+// TODO: sourcing from JSON for demo purposes.
+import TEMP_CONTENT from './content.json'
+
+const basePath = 'docs'
+const productName = 'Waypoint'
+const productSlug = 'waypoint'
+
+async function getStaticProps(): Promise<$TSFixMe> {
+  const CONTENT = TEMP_CONTENT as $TSFixMe
+
+  // TODO: Content blocks content should likely
+  // TODO: be fetched dynamically, currently things
+  // TODO: like the title and description of docs pages
+  // TODO: are hard-coded alongside the URL to link to.
+  // TODO: Should be possible to fetch the title and description
+  // TODO: at build time from said URL, to ensure the displayed
+  // TODO: content is accurate, while not needing to be
+  // TODO: manually kept up to date?
+  CONTENT.blocks = CONTENT.blocks.map((block) => {
+    const { type, heading } = block
+    if (type !== 'heading') return block
+    return {
+      ...block,
+      slug: slugify(heading, { lower: true }),
+    }
+  })
+
+  const navData = [
+    { title: 'Introduction', href: '/waypoint/docs/intro' },
+    { title: 'Getting Started', href: '/waypoint/docs/getting-started' },
+    { divider: true },
+    { heading: 'Resources' },
+    {
+      title: 'Releases',
+      href: 'https://releases.hashicorp.com',
+      _demoExternalLink: true,
+    },
+    {
+      title: 'HashiCorp Learn',
+      href: 'https://learn.hashicorp.com',
+      _demoExternalLink: true,
+    },
+    {
+      title: 'Community Forum',
+      href: 'https://discuss.hashicorp.com',
+      _demoExternalLink: true,
+    },
+    {
+      title: 'Support',
+      href: 'https://support.hashicorp.com',
+      _demoExternalLink: true,
+    },
```

Comment:
Not sure if you've seen this yet or not, but it looks like these links are showing at the top of the sidebar in the docs, plugins, and cli pages after merging in my sidebar changes.

![image](https://user-images.githubusercontent.com/43934258/148124939-3447f69f-72e6-4982-9904-c3f84a9f16dd.png)

Reply:
Thanks for calling this out! Have updated slightly, ended up being a little less dry, but have the `/waypoint` page sidebar contents displaying 👍